pip/cmdoptions.py

```diff
@@ -498,6 +498,18 @@ def only_binary():
          "options to setup.py install. If you are using an option with a "
          "directory path, be sure to use absolute path.")
 
+download_options = partial(
```

Comment:
Yes I saw it was declared but I didn't see where it was used by pip (no `options.dowload_options` used anywhere).

For `pip install`, it is used in https://github.com/pypa/pip/blob/develop/pip/commands/install.py#L208 and those options are passed down to the final `setup.py install` call when installing.